web/components/application/hacker/Hacker.sass

```diff
@@ -0,0 +1,27 @@
+@import ./pages/PageOne
+
+div#hacker-application
```

Comment:
any particular reason to use an ID selector? It might be best if we try to abstract this out a bit into a class - I imagine that this layout might be something we'll use again for another component, as it's pretty general. Would help keep things looking uniform as well.

I've been thinking about setting up sass-lint for this project - set it up for another project recently, and it expressly forbids ID selectors, probably for this reason

Reply:
I get why we should use a class here (because other components like `volunteer-application` and `mentor-application` would use it too) but I'm not sure why we should strictly forbid `id` selectors everywhere else?

Reply:
I only bring it up because it's a default rule, but I think it makes sense - why ID-select, thereby restricting potentially useful CSS to only one element, when a sufficiently unique class name will serve the same purpose, while leaving the door open for reusability?

This StackOverflow thread presents some arguments on this topic: https://stackoverflow.com/questions/8279132/why-shouldnt-i-use-id-selectors-in-css, while the top comment supports the use of ID selectors, imo the arguments against are more compelling.